source/content/cms-admin.md

```diff
@@ -1,231 +1,283 @@
-```
-
-## Commit SFTP Changes
-While in SFTP Mode, changes made to the codebase are "staged", as visible on in the <span style="line-height:.9" class="glyphicons glyphicons-wrench"></span> **Dev** tab:
-
-![Pending changes in SFTP mode](../images/dashboard/dev-code-pendingchanges-commit.png)
-
-Changing your site's connection mode from SFTP to Git will discard all uncommitted file changes. If you want to keep work in progress, commit before toggling the connection mode.
-
-## Troubleshooting
-
-### I can't write to my codebase on Test or Live.
-
-This is by design. Please read [this section](/pantheon-workflow#understanding-write-permissions-in-test-and-live) of our Pantheon Workflow article to understand why.
-
-### Unable to Install Plugins on Migrated Sites
-
-On some WordPress sites migrated from other platforms, when installing or updating plugins you may see an error like this:
-
-![Failed Plugin Installation](../images/wp-plugin-failed.png)
-
-This is usually caused by the following lines in `wp-config.php`, added by your former host:
-
-```
-define('FS_CHMOD_DIR', 755);
-define('FS_CHMOD_FILE', 644);
-```
-
-Remove these lines to resolve.
-
-A `wp-content/upgrade` folder with read-only access can also cause this error. You can remove this folder or change the file permission to `755` to make it writable. WordPress stores temporary files to the `wp-content/upgrade` folder when updating plugins or themes.
-
-### Unable to Install Drupal Module or Theme
-
-There is a known bug in Drupal 8 impacting the **+ Install new module** and **+ Install new theme** buttons. See [Modules With Known Issues](/modules-known-issues#update-manager) for details.
+---
+title: Working in the WordPress Dashboard and Drupal Admin Interface
+description: Learn how to build sites using the WordPress and Drupal admin interfaces in SFTP mode on Pantheon.
+searchboost: 150
+categories: [develop]
+tags: [site, sftp, dashboard]
+reviewed: "2020-06-24"
+---
+Pantheon's Site Dashboard provides two connection modes to support various development workflows, such as pushing commits from your local with [Git](/git) or working in the WordPress or Drupal admin interface in [SFTP](/sftp) mode. Admin tools and command-line interfaces require write access to the codebase, which is only provided to development environments (Dev or [Multidev](/multidev)) in **SFTP** mode.
+
+<Alert title="Exports" type="export">
+
+This doc offers [Terminus](/terminus) commands, using the variables `$site` and `$env`. Export these variables in your terminal session to match your site name and the correct environment:
+
+```bash{promptUser: user}
+export site=yoursitename
+export env=dev
+```
+
+</Alert>
+
+## SFTP Mode
+
+1. Navigate to the **<span class="glyphicons glyphicons-wrench"></span> Dev** or **<span class="glyphicons glyphicons-cloud"></span> Multidev** tab of your Pantheon Site Dashboard.
+1. Next to Connection Mode, click **SFTP**:
```

Comment:
looks like this should be item #2 if I'm reading it right

Reply:
Markdown increments 1'd numbered steps, and we follow that convention so that the addition or removal of a step doesn't necessitate changing all the others.